BraintreeDropIn/BTPaymentSelectionViewController.m

```diff
@@ -187,9 +187,16 @@ - (void)configurationLoaded:(__unused BTConfiguration *)configuration error:(NSE
                 }
             }
 
-            // Always add Cards
-            [activePaymentOptions addObject:@(BTUIKPaymentOptionTypeUnknown)];
-            
+            NSArray *supportedCardTypes = [self.configuration.json[@"creditCards"][@"supportedCardTypes"] asArray];
+            for (NSString *supportedCardType in supportedCardTypes) {
+                BTUIKPaymentOptionType paymentOptionType = [BTUIKViewUtil paymentOptionTypeForPaymentInfoType:supportedCardType];
+                if ([BTUIKViewUtil isPaymentOptionTypeACreditCard:paymentOptionType]) {
```

Comment:
I guess a better question is, is there anything that we would get back in that `supportedCardTypes` array that's not a credit card?